### DISCUSSION TITLE
Sessions recording

### DISCUSSION THREAD

**Discussion by itishermann** [plausible/analytics#978] [Anything Else]
Not sure if it’s something that fits in with Plausible but it would be nice to have a card on the dashboard to record sessions and maybe build heatmaps.
So people like me can forget Microsoft Clarity or rrweb 😁

**Comment by metmarkosaric** [comment]
Thanks! We do not have the intention to add heatmaps and session recordings. It's not really in the scope of the project as we'd like to stay simple, lightweight and privacy first.
